dev-libs/libmodbus/libmodbus-3.0.8.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="A groovy modbus library"
+DESCRIPTION="libmodbus is a free software library to send or receive data with a \
+device which respects the Modbus protocol. This library can use a serial port or \
+an Ethernet connection.
+
+The functions included in the library have been derived from the Modicon Modbus  \
+Protocol Reference Guide which can be obtained from www.modbus.org.
+"
```

Comment:
Not quite right here, check https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description

Reply:
ok done